tests/test_visitors/test_ast/test_classes/test_methods/test_useless_overwriting_method.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+
+from typing import List
+
+import pytest
+
+from tests.test_visitors.test_ast.test_classes.test_methods.data_model import (
```

Comment:
We disallow imports from `tests`, for some reason `layers.yml` misses that.
Can you please:
1. Move it to `pytest` fixtures
2. Move into this module

Reply:
Btw, why not just tuples?

Reply:
I will go for tuples

Reply:
Maybe I could use there at least namedtuples?)

Reply:
NamedTuples is fine 👍